samples/matter/weather_station/README.rst

```diff
@@ -0,0 +1,154 @@
+Both methods can be enabled after :ref:`building and running the sample <matter_lock_sample_remote_control>`.
+
+.. _matter_weather_station_network_mode:
+
+Remote testing in a network
+===========================
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_remote_testing_start
+    :end-before: matter_door_lock_sample_remote_testing_end
+
+.. _matter_weather_station_test_mode:
+
+Remote testing using test mode
+==============================
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_test_mode_start
+    :end-before: matter_door_lock_sample_test_mode_end
+
+Configuration
+*************
+
+|config|
+
+Device Firmware Upgrade support
+===============================
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_build_with_dfu_start
+    :end-before: matter_door_lock_sample_build_with_dfu_end
+
+User interface
+**************
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_button1_start
+    :end-before: matter_door_lock_sample_button1_end
+
+Serial Wire Debug port:
+    Used for getting logs from the device or communicating with it through the command-line interface.
+
+NFC port with antenna attached:
+    Optionally used for obtaining the commissioning information from the Matter accessory device to start the :ref:`commissioning procedure <matter_weather_station_sample_remote_control_commissioning>`.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/matter/weather_station`
+
+.. include:: /includes/build_and_run.txt
+
+See `Configuration`_ for information about building the sample with the DFU support.
+
+Testing
+=======
+
+You can either test the sample's :ref:`basic features <matter_light_bulb_sample_basic_features_tests>` or use the light switch sample to test the light bulb's :ref:`communication with another device <matter_light_bulb_sample_light_switch_tests>`.
+
+.. _matter_weather_station_sample_basic_features_tests:
+
+Testing basic features
+----------------------
+
+After building the sample and programming it to your development kit, test its basic features by completing the following steps:
+
+#. |connect_kit|
+#. |connect_terminal|
+#. Press **Button 1** to initiate the factory reset of the device.
+
+.. _matter_weather_station_sample_remote_control:
+
+Enabling remote control
+=======================
+
+Remote control allows you to control the Matter light bulb device from a Thread network.
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_remote_control_start
+    :end-before: matter_door_lock_sample_remote_control_end
+
+.. _matter_weather_station_sample_remote_control_commissioning:
+
+Commissioning the device
+------------------------
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_commissioning_start
```

Comment:
Generally that section should work fine for Thingy with one difference: ` ... a QR code, printed to the UART console ...` - Thingy:53 doesn't have UART console. It uses RTT interface.